cfgov/v1/migrations/0012_share_perms.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+def create_share_permissions(apps, schema_editor):
+    ContentType = apps.get_model('contenttypes.ContentType')
+    Permission = apps.get_model('auth.Permission')
+    Group = apps.get_model('auth.Group')
+
+    # Add a fake content type to hang the 'can access Wagtail admin' permission off.
+    # The fact that this doesn't correspond to an actual defined model shouldn't matter, I hope...
+    v1_content_type = ContentType.objects.create(app_label="v1", model="cfgovpage")
+
+    # Create share permission
+    share_permission = Permission.objects.create(
+        content_type=v1_content_type,
+        codename='share_page',
+        name='Can share pages'
+    )
+
+    # Assign it to Editors and Moderators groups
+    for group in Group.objects.filter(name__in=['Editors', 'Moderators']):
```

Comment:
lol gotcha, yea i figure it doesn't hurt if they add more default groups in future releases